client/src/components/StepWizard/WizardNav.js

```diff
@@ -32,38 +32,24 @@ const PrevButton = styled(Button)`
   border-color: transparent;
 `;
 
-const CircleButton = styled(Button)`
-  justify-self: flex-end;
-  background-color: ${PRIMARY};
-  border-radius: 24px;
-`;
-
 const WizardNav = ({ currentStep, nextStep, previousStep, totalSteps }) => (
   <StyledWizardNav>
     {currentStep > 1 ? (
       <PrevButton
         plain
         onClick={previousStep}
-        icon={<FormPreviousLink />}
+        icon={<BackArrow />}
         a11yTitle={`Navigate to step ${currentStep - 1}`}
       />
     ) : (
       <PrevButton
         plain
         href="/"
-        icon={<FormPreviousLink />}
+        icon={<BackArrow />}
         a11yTitle="Navigate to the homepage"
       />
     )}
-    {currentStep < totalSteps && (
-      <CircleButton
-        primary
-        color="primary"
-        onClick={nextStep}
-        a11yTitle={`Navigate to step ${currentStep + 1}`}
-        icon={<FormNextLink color="#fff" />}
```

Comment:
@robinv85 Yea so there weren't any bright fluorescent green borders but it came back when I replaced the icon value with the SVG `<NextArrow />`